recipes/ucsc-gff3togenepred/include.patch

```diff
@@ -5,7 +5,7 @@
  
  HG_DEFS=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_GNU_SOURCE -DMACHTYPE_${MACHTYPE}
 -HG_INC+=-I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc
-+HG_INC+=-I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc -I /anaconda/include
++HG_INC+=-I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc -I /anaconda/include -I ${PREFIX}/include
```

Comment:
Do we really need the /anaconda/include? I would prefer the recipe to be independent of our setup. I think $PREFIX/include should be sufficient.